src/main/fabric8/configmap.yml

```diff
@@ -0,0 +1,4 @@
+data:
+  application.properties: |
```

Comment:
Please mention this config support in the readme

Reply:
Fixed in upcomming commit